internal/ingress/annotations/sessionaffinity/main.go

```diff
@@ -75,6 +81,10 @@ type Cookie struct {
 	Path string `json:"path"`
 	// Flag that allows cookie regeneration on request failure
 	ChangeOnFailure bool `json:"changeonfailure"`
+	// SameSite attribute value
+	SameSite string `json:"samesite"`
+	// Flag that conditionally applies SameSite=None attribute on cookie if user agent accepts it.
+	ConditionalSameSiteNone bool `json:"conditional_samesite_none"`
```

Comment:
conditional_samesite_none -> conditional-samesite-none

Reply:
This may be a dumb question, but how does this relate to... the `CookieSessionAffinity` type struct in `internal/ingress/types.go` line 146ish?